docroot/profiles/custom/cgov_site/modules/custom/cgov_biography/config/install/core.entity_view_display.node.cgov_biography.default.yml

```diff
@@ -59,6 +60,16 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
+  field_bio_press_info:
```

Comment:
Please change this to be a dropdown select (Do Not Display vs Display). It should be an integer list with Display being 1 and Do not being 0. @VictoriaSunNIH Can you please update the requirements.  (Checkboxes get hidden in the UI.)

Also can you make the field name something like field_display_bio_press_info? So it is more of a question.